src/averageAges.js

```diff
@@ -15,11 +15,20 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let menInArray = people.filter(person => person.sex === 'm');
+  const menDied = menInArray.filter(person => Math.ceil(person.died / 100)
+  === century);
+
+  menInArray = century
+    ? menDied
+    : menInArray;
```

Comment:
do you need to make 2 filters? you can check 'century' firstly and then decide which filter to apply (the same in next functions)